types/utils.d.ts

```diff
@@ -3,6 +3,10 @@ declare global {
         [K in keyof T]-?: T[K] extends V ? K : never;
     }[keyof T];
 
+    type ObjectEntries<T> = {
```

Comment:
@yauheni-deriv we won't need this type because TS can infer (recognize) the type of Object.entries() on its own :)

Reply:
@yauheni-deriv then why do I get no TS errors when I remove this type from the only place where it's used in packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx? :)
<img width="763" alt="Screenshot 2023-08-21 at 4 54 57 PM" src="https://github.com/binary-com/deriv-app/assets/103177211/e898af38-8085-4349-88fc-77e3ee14fc74">

Reply:
Because there is ts-nocheck comment in the top of the file.
 I mentioned it in file-uploader.tsx as well. There are dependencies of TS issues across different files. We will refactor it in separate PR

Reply:
ha, missed that :)) please ignore)